my-app/src/home.js

```diff
@@ -0,0 +1,97 @@
+import React from "react"
+
+import '../node_modules/bootstrap/dist/css/bootstrap.min.css';
+import Dropdown from "./dropDown"
+import PopUp from "./popup"
+import Help from "./help"
+import drag from "./drag.png"
+import Grid from "./grid"
+import History from "./history"
+
+let folders = ["folder1", "folder2", "folder3"]
+
+class Home extends React.Component{
+	
+
+	render(){
+		return(
+			<div>
+				Welcome to Home page~
+				<nav class="navbar navbar-expand-lg navbar-light bg-light">
+				  <a class="navbar-brand" href="/">Collaboard</a>
```

Comment:
Clicking "Collaboard" when the user is logged in should not take them back to the login page.

Also, need a logout button